pkg/acctest/test.go

```diff
@@ -0,0 +1,160 @@
+package acctest
```

Comment:
This file contains mostly of duplicated code from the Artifactory repo. We need to move these code to the shared repo and then update each provide repos to use that instead.